configure.py

```diff
@@ -1948,10 +1945,11 @@ def configure_section_file(o):
   except OSError:
     if options.node_section_ordering_info != "":
       warn('''No acceptable ld.gold linker found!''')
-    return 0
```

Comment:
Please revert.  If this function has no type hints, then we should not guess what function callers are expecting to have returned.  Use `#noqa` if needed.